### PR TITLE
fix: Missing translations

### PR DESCRIPTION
**Pull request by memoyil**

### CODE REVIEW

File: src/views/Farms/components/FarmTable/Farm.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const Farm: React.FunctionComponent<FarmProps> = ({ image, label, pid }) => {
     if (rawStakedBalance) {
       return (
         <Text color="secondary" fontSize="12px" bold>
-          {t('FARMING')}
+          {t('Farming').toUpperCase()}
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Farm.tsx:43]: text transform

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const HarvestAction: React.FunctionComponent<HarvestActionProps> = ({ pid, userD
     <ActionContainer>
       <ActionTitles>
         <Title>CAKE </Title>
-        <Subtle>{t('EARNED')}</Subtle>
+        <Subtle>{t('Earned').toUpperCase()}</Subtle>
```
Comment (hachiojidev) [inline src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:52]: The `<Subtle />` component is not necessary. Don't worry about changing it now so for this case we can keep `toUpperCase`. However, can you add a `TODO:` for some dev to remove this component in favor of `<Text />`

File: src/views/Ifos/components/IfoFoldableCard/index.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ const getRibbonComponent = (status: IfoStatus, t: any) => {
   }
 
   if (status === 'live') {
-    return <CardRibbon variantColor="primary" ribbonPosition="left" text={t('LIVE!')} />
+    return <CardRibbon variantColor="primary" ribbonPosition="left" text={`${t('Live').toUpperCase()}!`} />
```
Comment (hachiojidev) [inline src/views/Ifos/components/IfoFoldableCard/index.tsx:34]: In this case we should use the `style` prop to transform the text.

File: src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx [hunk 4]
```diff
@@ -104,7 +104,7 @@ const ExpandedFooter: React.FC<ExpandedFooterProps> = ({ pool, account, isAutoVa
               <Skeleton width="54px" height="21px" />
             )}
             <Text ml="4px" color="primary" small>
-              {t('blocks')}
+              {t('Blocks').toLowerCase()}
```
Comment (hachiojidev) [inline src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx:107]: Text transform

**Review by hachiojidev — CHANGES_REQUESTED** [review]
Thank you for going through and doing this. It couldn't have been fun. Other than the small comments I left can you test that the `t` function when passed as a dependency in an effect doesn't cause infinite re-renders.

File: src/views/Farms/Farms.tsx [hunk 5]
```diff
@@ -388,35 +388,35 @@ const Farms: React.FC = () => {
           </ViewControls>
           <FilterContainer>
             <LabelWrapper>
-              <Text>SORT BY</Text>
+              <Text>{t('Sort by').toUpperCase()}</Text>
```
[the remark this answers is not in the document]
Comment (memoyil) [reply src/views/Farms/Farms.tsx]: Done

File: src/views/Farms/Farms.tsx [hunk 6]
```diff
@@ -388,35 +388,35 @@ const Farms: React.FC = () => {
...
+                    label: t('Liquidity'),
                     value: 'liquidity',
                   },
                 ]}
                 onChange={handleSortOptionChange}
               />
             </LabelWrapper>
             <LabelWrapper style={{ marginLeft: 16 }}>
-              <Text>SEARCH</Text>
+              <Text>{t('Search').toUpperCase()}</Text>
```
[the remark this answers is not in the document]
Comment (memoyil) [reply src/views/Farms/Farms.tsx]: Done

File: src/views/Farms/components/FarmTable/Farm.tsx (hunk 1 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Farm.tsx]: Done

File: src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx (hunk 2 above)
Comment (memoyil) [reply src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx:52]: I added TODO

File: src/views/Ifos/components/IfoFoldableCard/index.tsx (hunk 3 above)
Comment (memoyil) [reply src/views/Ifos/components/IfoFoldableCard/index.tsx]: Done

File: src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx (hunk 4 above)
Comment (memoyil) [reply src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx]: Done

**Comment by memoyil** [comment]
@hachiojidev Checked t as a dependency in useEffect and it works properly

**Review by hachiojidev — APPROVED** [review]

**Pull request merged**